src/proxyvisor.coffee

```diff
@@ -146,7 +146,7 @@ class ProxyvisorRouter
 			.then ([ device ]) =>
 				return res.status(404).send('Device not found') if !device?
 				return res.status(410).send('Device deleted') if device.markedForDeletion
-				@logger.log(m, { channel: "device-#{device.logs_channel}-logs" })
+				@logger.log(m, { uuid, channel: "device-#{device.logs_channel}-logs" })
```

Comment:
Why aren't we using `logDependent` here? Ideally we need dep devices to use the API logging.

Reply:
I've tested this locally, and this needs to be changed to `@logger.logDependent(m, { uuid })` otherwise the log message will be sent using the gateway device's UUID

Reply:
Yeah, good catch, I forgot to change it when I introduced the logDependent function. Will fix.

Reply:
Done